src/cfnlint/custom_rules.py

```diff
@@ -0,0 +1,53 @@
+import logging
+import cfnlint
+
+
+LOGGER = logging.getLogger(__name__)
+Operators = {'EQUALS': lambda x, y: equals(x, y)}
+
+def check_custom_rules(filename, template):
+    """ Process custom rule file """
+    matches = []
+    with open(filename) as customRules:
+        count = 1
+        for line in customRules:
+            LOGGER.debug('Processing Custom Rule Line %d', count)
+            line = line.replace('"', '')
+            rule = line.split(' ')
+            if len(rule) == 4 and rule[0][0] != '#':
+                try:
+                    result = Operators[rule[2]](rule, template.get_resource_properties([rule[0]]))
+                    if result != rule[2]:
+                        matches.append(cfnlint.rules.Match(count, '0', '0', '0', filename, CustomRule('E9999'), result, None))
+                except KeyError as e:
+                    matches.append(cfnlint.rules.Match(count, '0', '0', '0', filename, CustomRule('E9999'), 'Error - Invalid Operator', None))
+            count += 1
+    return matches
+
+def equals(rule, propertyList):
```

Comment:
To keep things organized, i think this operator code should be in a separate class

Reply:
Moved to a separate folder - Keeping it in one "Operator" class for now as I do feel each individual operator functions will likely remain small and neat. If not, it should be trivial to separate out